quine/src/main/scala/com/thatdot/quine/app/importers/package.scala

```diff
@@ -213,6 +227,17 @@ package object importers extends StrictLogging {
         )
     }
 
+  private def getTranscoder(charsetName: String): (Charset, Flow[ByteString, ByteString, NotUsed]) =
```

Comment:
Could you migrate the doc above the call site of `getTranscoder` to be a Scaladoc? It seems relevant to the function in general.